### ISSUE TITLE
Monitoring multiple servers

### ISSUE THREAD

**Issue by edgrosvenor** [laravel/telescope#62]
I'm enjoying this so far. Odd thing, though. Telescope seems to have started an exclusive relationship with just one of my servers and isn't on speaking terms with the others. The server being monitored happens to be the one on which I ran the migration, though I don't know whether that's in any way relevant. I use Envoyer to deploy to 4 servers. Is there a configuration I'm missing that would let me monitor them all? 

I apologize for taking the easy way out and asking the question rather than digging through the code looking for an answer. But I suspect I'm not the only person who will have this question.

**Comment by themsaid** [comment]
If Telescope is installed on all the servers and configured to use the same database connection, it should record the entries from all the servers no problem. You'll need to make sure it's installed on the other servers.

**Comment by edgrosvenor** [comment]
@themsaid Thank you. I sort of see what happened. Somehow on 3 out of 4 of my servers, the php artisan telescope:install command failed, saying nothing existed in the telescope namespace. It did appear that something went a little wrong when I deployed. I had written a one time deployment hook that was supposed to do the installation on all 4 servers before the migration, which obviously runs on a single server. 

Anyway, I've published the files on all the servers and will see what happens. I have a handful of installations still to do on multi-server Envoyer-backed setups so if I find anything quirky in the process, I'll throw you some documentation or feedback.

Thank you!
